build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'com.indeed.oss' version '1+'
+    id 'maven-publish'
+    id 'java'
+}
+
+indeedOss.activateFeature 'library'
+indeedLibrary.name = 'proctor-parent'
+
+publishing {
```

Comment:
This whole publishing section is not needed and is accounted for by the library feature activated above.

Reply:
I added this for always using 9999-snapshot as the local install to be able to easily test local publishes. 

Is there a better way to accomplish this? 

For more info: in proctor internal we either get latest or set to version 9999-snapshot to test local publish. I added this to maintain this legacy feature.

Reply:
Will move forward with manually copying the version